### ISSUE TITLE
Include selenium Keys with py?

### ISSUE THREAD

**Issue by chingc** [ElSnoMan/pyleniumio#188]
I saw the following in the docs for `type` and was wondering if `Keys` could just be part of `py` so it doesn't need to be imported.  It could be accessed by `py.Keys` or something similar.  It'd be a small but nice quality of life improvement.

```
# import the Keys from selenium
py.get('#search').type('puppies', Keys.ENTER)
```

Also, is it possible to silence the webdriver manager output?

```
====== WebDriver manager ======
Current google-chrome version is 90.0.4430
Get LATEST driver version for 90.0.4430
```

For runs that have many tests I see this throughout my output and it gets very noisy.

Thanks!

**Comment by ElSnoMan** [comment]
That should be a pretty quick add to the `Pylenium` class. Good idea!

As for the webdriver manager, I don't know how to suppress those logs cuz I've never had a need to. Usually you want that kind of info for traceability so we've always included it

**Comment by chingc** [comment]
Thank you!  Looking forward to the next release!

**Comment by ElSnoMan** [comment]
@chingc It's already released! Check it out with v1.12.4

**Comment by chingc** [comment]
I figured out a way to silence the webdriver logs with an automatic fixture.  :)

```
@pytest.fixture(autouse=True)
def no_wdm_logs():
    """Silence webdriver-manager logs.

    This is the stdout output from Pylenium that shows the use/download
    of browser drivers.  Change "autouse" to False to display its logs.
    """
    os.environ['WDM_LOG_LEVEL'] = '0'
```
